array_api_strict/_dtypes.py

```diff
@@ -121,7 +121,7 @@ def __hash__(self):
     "integer": _integer_dtypes,
     "integer or boolean": _integer_or_boolean_dtypes,
     "boolean": _boolean_dtypes,
-    "real floating-point": _floating_dtypes,
+    "real floating-point": _real_floating_dtypes,
```

Comment:
Good catch. Is this covered by one of the new tests?

Reply:
Not explicitly. I found this because a test was failing, looked into this and found it. But also it is too long ago for me to remember what exactly it was :-/